terraform/modules/len/lambdas/len.py

```diff
@@ -49,11 +49,9 @@ def lambda_handler(event, context):
 
 
 def run_scheduled_job(auth_token: str):
-    now = datetime.now()
-    minutes_into_day = (now - now.replace(hour=9, minute=0, second=0)).total_seconds() / 60
-    number_of_events = math.floor((100 / (3 * math.pi)) * math.exp(-0.5 * (((minutes_into_day / 60) - 4.5) / 1.5) ** 2))
+    number_of_events = 5
 
-    for i in range(number_of_events):
+    for _i in range(number_of_events):
```

Comment:
was this purposeful

Reply:
Yes, it's the standard style for unused variables. Stops my IDE from shouting at me